src/Utilitywarehouse.OpsEndpoints/Owin/OpsEndpointsMiddleware.cs

```diff
@@ -21,14 +21,12 @@ public class OpsEndpointsMiddleware
         private readonly RequestDelegate _next;
         private readonly JsonSerializer _serializer;
         private readonly Dictionary<string, HttpHandler> _handlers;
+        private readonly JsonSerializerSettings _serializerSettings;
```

Comment:
It looks like these are not being used anywhere in the file, same as the JsonSerializer. Are these configured globally somewhere?